chronicle-queue/src/test/scala/akka/stream/alpakka/chroniclequeue/StreamSpecUtil.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+// ORIGINAL LICENCE
+/*
+ *  Copyright 2017 PayPal
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package akka.stream.alpakka.chroniclequeue
+
+import java.io.File
+import java.nio.file.Files
+import java.util.concurrent.atomic.AtomicInteger
+
+import akka.stream.ThrottleMode
+import akka.stream.scaladsl._
+import com.typesafe.config.ConfigFactory
+import net.openhft.chronicle.wire.{WireIn, WireOut}
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+import scala.collection.JavaConverters._
+import scala.util.Random
+
+import akka.stream.alpakka.chroniclequeue.impl.Event
+
+object StreamSpecUtil {
```

Comment:
Util classes are dumping grounds. Can we search for a better name?

Reply:
made an attempt, if you have a better naming feel free to propose!